test/exercism/user_finished_tracks_test.rb

```diff
@@ -14,6 +14,7 @@ def setup
     create_exercise('fake', 'two')
     create_exercise('jewels', 'hello-world')
     create_exercise('jewels', 'gemerald')
+    create_exercise('shoes','Hello-world')
```

Comment:
This test here is to assert that a problem that is not present in the track is not counted, right? I got confused for a second between `Hello-world` and `hello-world`.  What do you think of replacing `Hello-world` with something like `deprecated-problem`? Let me know if my assumption is wrong. :)